File: level80/homework/codwars9.js
//https://www.codewars.com/kata/51e04f6b544cf3f6550000c1/train/javascript
// Returns number of complete beeramid levels
var beeramid = function(bonus, price) {
  var beers = Math.floor(bonus / price), levels = 0;
  while(beers >= ++levels * levels) {
    beers -= levels * levels;
  }
  return levels - 1;
}

File: level80/homework/codwars9.py
#https://www.codewars.com/kata/51e04f6b544cf3f6550000c1/train/python
def beeramid(bonus, price):
    beers  = bonus // price
    levels = 0
    
    while beers >= (levels + 1) ** 2:
        levels += 1
        beers  -= levels ** 2
    
    return levels